AdyenSession/API/Payments/PaymentsRequest.swift

```diff
@@ -51,6 +51,8 @@ internal struct PaymentsRequest: APIRequest {
         try container.encodeIfPresent(data.socialSecurityNumber, forKey: .socialSecurityNumber)
         try container.encodeIfPresent(data.browserInfo, forKey: .browserInfo)
         try container.encodeIfPresent(data.order?.compactOrder, forKey: .order)
+//        try container.encode("iOS", forKey: .channel)
+//        try container.encode(data.amount, forKey: .amount)
```

Comment:
why this is commented?

Reply:
Those don't exist in these new requests, will talk these as we go with backend team